Resources/config/security_expressions.xml

```diff
@@ -27,7 +27,7 @@
 
     <services>
         <service id="security.extra.twig_extension" class="%security.extra.twig_extension.class%" public="false">
-            <argument type="service" id="security.context" />
+            <argument type="service" id="security.authorization_checker" />
```

Comment:
IMO, you should use a compiler pass [like in the ``FOSRestBundle``](https://github.com/FriendsOfSymfony/FOSRestBundle/blob/1.8/DependencyInjection/Compiler/ParamFetcherConfigurationPass.php#L25) to avoid dropping symfony 2.3 support.